docs/src/main/asciidoc/elytron-properties-guide.adoc

```diff
@@ -0,0 +1,168 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Elytron Properties File Config
+
+include::./attributes.adoc[]
+
+Quarkus provides support for properties file based authentication that is intended for
+development and testing purposes. It is not recommended that this be used in production as at present only
+plaintext passwords are supported.
```

Comment:
This is not true anymore?

Reply:
Oops, I wrote this before I decided I was going to go ahead and implement the hashed passwords